org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/GridTabRowRenderer.java

```diff
@@ -650,7 +660,7 @@ else if (DisplayType.isNumeric(gridPanelFields[i].getDisplayType())) {
 				}
 				
 				GridRowCtx ctx = new GridRowCtx(Env.getCtx(), gridTab, rowIndex);
-				if (! (gridPanelFields[i].isDisplayed(ctx, true) || gridPanelFields[i].isDisplayedGrid())){
+				if (! (gridPanelFields[i].isDisplayed(ctx, true) || gridPanelFields[i].isDisplayedGrid(ctx, true))){
```

Comment:
Don't think you need to call isDisplayed(ctx, true) here, it is mostly the same as isDisplayedGrid(ctx, true).

Reply:
Yes, here isDisplayedGrid() method we checked based on the **IsDisplayedGrid** flag is only different and it's only in grid mode